aspnet/config-builder.md

```diff
@@ -13,13 +13,16 @@ msc.type: content
 
 By [Stephen Molloy](https://github.com/StephenMolloy) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-Configuration builders provide a mechanism for ASP.NET apps to get configuration values from external sources.
+Configuration builders provide a modern and agile mechanism for ASP.NET apps to get configuration values from external sources.
 
 Configuration builders:
 
 * Are available in .Net Framework .Net 4.7.1 and later.
 * Provide a flexible mechanism for reading configuration values.
 * Address some of the basic needs of apps as they move into a container and cloud focused environment.
+* Operate on two sections, `<appSettings />` and `<connectionStrings />`.
```

Comment:
"Configuration Builders" as a concept - the concept that is introduced in .Net 4.7.1 - apply to any config section. Not just appSettings and connectionStrings.

They set of key/value config builders that we ship via nuget are paritally constrained to those two sections. In token-replace mode (mode="Expand") can be used on any config section.